assets/js/directives/scroll-in-view.directive.js

```diff
@@ -9,6 +9,7 @@ function scrollInView($window, $timeout) {
   const directive = {
     restrict: 'E',
     templateUrl: 'templates/scroll-in-view.jade',
+    replace: true,
```

Comment:
It seemed like have <scroll-in-view><div class="scroll-in-view"> was a unnecessary.